e2e/single-cluster/status_test.go

```diff
@@ -107,3 +117,175 @@ var _ = Describe("Checks status updates happen for a simple deployment", Ordered
 		})
 	})
 })
+
+var _ = Describe("Checks that template errors are shown in bundles and gitrepos", Ordered, Label("infra-setup"), func() {
```

Comment:
What prevents these tests from being integration tests instead?

Reply:
I'm not actually sure that there is something that would prevent that, I was just worried about the complexity of having to work with several controllers in integration tests.

Reply:
We already [support](https://github.com/rancher/fleet/blob/main/integrationtests/gitjob/controller/suite_test.go#L107-L133) multiple reconcilers in integration tests ;)